scripts/mobile-service-stack/README.md

```diff
@@ -0,0 +1,41 @@
+# Process to Run Developer Builds on Mobile Devices
+This process is relevant for viewing the Medic webapp on mobile devices when the api service is run on a developer machine running Webapp >v3.5.0. Webapp v3.5.0 relies on service workers, which require a valid HTTPS certificate to function. Follow these steps to make your developer build accessible from your android device at the trusted url https://medic.dev.
+
+## Preparing (do this once)
+### Setup for an android devices
+Perform these steps **once** for each Android device you wish to use:
+
+1. Install the Medic Mobile Android App by following [these installation steps](https://github.com/medic/medic-android#installation) or downloading it from the app store.
+1. Install Medic's developer certificate as a trusted authority.
+  a. Download [this certificate](https://raw.githubusercontent.com/medic/medic/tree/master/scripts/mobile-service-stack/https/root-ca.crt).
```

Comment:
This link isn't working for me. 

https://raw.githubusercontent.com/medic/medic/tree/master/scripts/mobile-service-stack/https/root-ca.crt

Reply:
The content is included in this PR -- so it will be broken until we merge to master. For now, you can use https://raw.githubusercontent.com/medic/medic/67cda1acfdc6f06aa19315b41eaaa06460aa08e8/scripts/mobile-service-stack/https/medic-ca.crt